packages/app-root/src/hooks/useThemeMode.js

```diff
@@ -0,0 +1,20 @@
+import { useState } from 'react'
+
+export default function useThemeMode(storedThemeMode) {
+  const [themeMode, setThemeMode] = useState(storedThemeMode)
+
+  function toggleTheme() {
+    let newTheme
+    if (themeMode === 'light') {
+      newTheme = 'dark'
+    } else {
+      newTheme = 'light'
+    }
+
+    setThemeMode(newTheme)
+    // The same key is used in PFE's theme mode toggle
+    document.cookie = `theme=${newTheme}`
```

Comment:
You probably want to set an expiry time here too. The `cookie` library can be useful to do that.
https://github.com/zooniverse/front-end-monorepo/blob/5f991b160e75de408279adf04cff183debf9e502/packages/app-project/stores/UI.js#L106-L110